elytron/EAP7-1856-Bearer-Token-Authorization-Timeout.adoc

```diff
@@ -0,0 +1,120 @@
+= [Preview] Bearer token timeout configurability will be added to WildFly's Elytron subsystem.
+:author:            Rebecca Searls
+:email:             rsearls@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:
+
+== Overview
+
+Bearer Token Authorization is the process of authorizing HTTP requests based on
+the existence and validity of a bearer token.  The token carries within it
+an expiration timestamp.  The two parameters being added, connectionTimeout
+and readTimeout are placed on the URL used in retrieving the public key.
+
+
+== Issue Metadata
+
+=== Issue
+* https://issues.redhat.com/browse/EAP7-1856[EAP7-1856]
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/ELY-2189[ELY-2189]
+* https://issues.redhat.com/browse/WFCORE-5691[WFCORE-5691]
+* https://issues.redhat.com/browse/EAPSUP-640[EAPSUP-640]
+
+=== Stability Level
+// Choose the planned stability level for the proposed functionality
+* [ ] Experimental
+
+* [x] Preview
+
+* [ ] Community
+
+* [ ] default
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+=== Testing By
+* [x] Engineering
+
+* [ ] QE
+
+=== Affected Projects or Components
+
+* WildFly-core
+* WildFly Elytron
+
+=== Other Interested Projects
+
+N/A
+
+=== Relevant Installation Types
+* [x] Traditional standalone server (unzipped or provisioned by Galleon)
+
+* [x] Managed domain
+
+* [x] OpenShift s2i
+
+* [x] Bootable jar
+
+== Requirements
+
+=== Hard Requirements
+
+* Two new attributes, `connectionTimeout` and `readTimeout` will be
```

Comment:
WildFly management API attributes should not be camel case, so these would be connection-timeout and read-timeout.

Reply:
done